### PR TITLE
chore: add more strings for translation

### LINKED ISSUE #7673: Some areas are not translatable
Please make the following areas translatable. If Weblate shows up 100% translated, but areas are not really translated something is wrong. Compare https://hosted.weblate.org/projects/open-event/frontend/de/

1. On wizard step 1 the offline payment options.
Screenshot from 2021-08-09 23-56-29

2. On the video tab the helper text
a) below the heading
b) on the (i) icon
Screenshot from 2021-08-09 23-57-31

3. Change "Video Room" to "Video" on the top area next to the switch

Screenshot from 2021-08-10 00-00-00

4. "No social links present" on the dashboard of events

Screenshot from 2021-08-10 00-24-05

5. On the groups edit pages
a) announce/remove event
Screenshot from 2021-08-10 00-26-43
b) Group join date
Screenshot from 2021-08-10 00-27-41
c) Settings page "About"
Screenshot from 2021-08-10 00-38-23

6. Ticket cancellation on tickets page - dialogue box

Screenshot from 2021-08-10 15-18-15

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #7673 

Added strings mentioned by @mariobehling

### CODE REVIEW

**Comment by progmatic-99** [comment]
If there are any more strings, you could find, mention here only. I will make changes in this PR only. @mariobehling

File: translations/en.po [hunk 1]
```diff
@@ -1105,12 +1105,8 @@ msgid "Your stream has been saved"
 msgstr "Your stream has been saved"
 
 #: app/components/forms/group/group-events-form.js:47:22
-msgid "Do you want to announce the event"
-msgstr "Do you want to announce the event"
-
-#: app/components/forms/group/group-events-form.js:47:100
-msgid "to all group members now"
-msgstr "to all group members now"
+msgid "Do you want to announce the event \"{{eventName}}\" to all group members now ?"
```
Comment (mariobehling) [inline translations/en.po:1108]: Do not show a space in front of "?"-

File: translations/es.po [hunk 2]
```diff
@@ -930,12 +929,16 @@ msgid "Room Password"
...
 
 #: app/components/forms/events/view/videoroom-form.hbs:78:19
+#: app/controllers/events/view/videoroom/list.js:59:21
 msgid ""
 "The room password field can be used to communicate a password which is "
 "necessary to access online video rooms for example for external video "
 "services such a Zoom, Teams and Webex. The need for a password entry depends "
 "on the configuration of your video channel. The integrated Big Blue Button "
-"video solution in"
+"video solution in {{appName}}.com does not need a password as only ticket "
```
Comment (mariobehling) [inline translations/es.po:938]: take out ".com"

File: translations/es.po [hunk 3]
```diff
@@ -930,12 +929,16 @@ msgid "Room Password"
 msgstr ""
 
 #: app/components/forms/events/view/videoroom-form.hbs:78:19
+#: app/controllers/events/view/videoroom/list.js:59:21
 msgid ""
 "The room password field can be used to communicate a password which is "
 "necessary to access online video rooms for example for external video "
 "services such a Zoom, Teams and Webex. The need for a password entry depends "
```
Comment (mariobehling) [inline translations/es.po:936]: change to "such as"

File: translations/es.po [hunk 4]
```diff
@@ -1123,11 +1126,9 @@ msgid "Your stream has been saved"
...
 
 #: app/components/forms/group/group-events-form.js:47:22
-msgid "Do you want to announce the event"
-msgstr ""
-
-#: app/components/forms/group/group-events-form.js:47:100
-msgid "to all group members now"
+msgid ""
+"Do you want to announce the event \"{{eventName}}\" to all group members "
+"now ?"
```
Comment (mariobehling) [inline translations/es.po:1131]: no space in front of "?"

File: translations/es.po [hunk 5]
```diff
@@ -3273,6 +3276,12 @@ msgid ""
 "the event page."
 msgstr ""
 
+#: app/controllers/events/view/videoroom/list.js:44:24
+msgid ""
+"This column shows the original link of the video. We do not recommend to "
+"share this link as users can access it without logging into {{appName}}.com."
```
Comment (mariobehling) [inline translations/es.po:3282]: take out .com

File: translations/es.po [hunk 6]
```diff
@@ -3273,6 +3276,12 @@ msgid ""
...
+msgstr ""
+
 #: app/controllers/events/view/videoroom/list.js:51:26
 msgid ""
 "You can share the links in this columns with attendees of an event. On event "
```
Comment (mariobehling) [inline translations/es.po:3287]: Correct sentence: "You can share the links in this columns with attendees of an event. Registered attendees or ticket holders will be able to access these links."

File: translations/fr.po [hunk 7]
```diff
@@ -965,12 +964,16 @@ msgid "Room Password"
...
 
 #: app/components/forms/events/view/videoroom-form.hbs:78:19
+#: app/controllers/events/view/videoroom/list.js:59:21
 msgid ""
 "The room password field can be used to communicate a password which is "
 "necessary to access online video rooms for example for external video "
 "services such a Zoom, Teams and Webex. The need for a password entry depends "
 "on the configuration of your video channel. The integrated Big Blue Button "
-"video solution in"
+"video solution in {{appName}}.com does not need a password as only ticket "
```
Comment (mariobehling) [inline translations/fr.po:973]: take out .com

File: app/components/forms/events/view/videoroom-form.hbs [hunk 8]
```diff
@@ -75,7 +75,7 @@
...
+          @content={{t 'The room password field can be used to communicate a password which is necessary to access online video rooms for example for external video services such a Zoom, Teams and Webex. The need for a password entry depends on the configuration of your video channel. The integrated Big Blue Button video solution in  {{appName}}.com does not need a password as only ticket holders are able to access it. The difference between the password and PIN is that the password option is used for online access while PINs are used to access video rooms through the telephone.' appName=this.settings.appName }} />
```
Comment (mariobehling) [inline app/components/forms/events/view/videoroom-form.hbs:78]: take out .com

File: app/controllers/events/view/videoroom/list.js [hunk 9]
```diff
@@ -41,7 +41,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       {
         name          : this.l10n.t('Video Source URL'),
         valuePath     : 'videoStream',
-        helperInfo    : this.l10n.t('This column shows the original link of the video. We do not recommend to share this link as users can access it without logging into') + this.settings.appName + '.com.',
+        helperInfo    : this.l10n.t('This column shows the original link of the video. We do not recommend to share this link as users can access it without logging into {{appName}}.com.', { appName: this.settings.appName }),
```
Comment (mariobehling) [inline app/controllers/events/view/videoroom/list.js:44]: take out .com

File: app/controllers/events/view/videoroom/list.js [hunk 10]
```diff
@@ -56,7 +56,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
...
+        helperInfo : this.l10n.t('The room password field can be used to communicate a password which is necessary to access online video rooms for example for external video services such a Zoom, Teams and Webex. The need for a password entry depends on the configuration of your video channel. The integrated Big Blue Button video solution in {{appName}}.com does not need a password as only ticket holders are able to access it. The difference between the password and PIN is that the password option is used for online access while PINs are used to access video rooms through the telephone.', { appName: this.settings.appName }),
```
Comment (mariobehling) [inline app/controllers/events/view/videoroom/list.js:59]: take out .com

**Comment by mariobehling** [comment]
Thanks. Please resolve merge conflicts.

**Review by mariobehling — APPROVED** [review]

**Pull request merged**